packages/foam-vscode/src/features/orphans.ts

```diff
@@ -1,8 +1,9 @@
 import * as vscode from 'vscode';
-import { Foam, FoamWorkspace, isNote, Resource } from 'foam-core';
+import { Foam, isNote, Resource } from 'foam-core';
 import { getOrphansConfig } from '../settings';
 import { FoamFeature } from '../types';
 import { GroupedResourcesTreeDataProvider } from '../utils/grouped-resources-tree-data-provider';
+import { FoamGraph } from 'packages/foam-core/src/model/workspace';
```

Comment:
Is this import path intentional?

Reply:
nope, I am also gonna pull graph in its own file